include/deal.II/numerics/fe_field_function.h

```diff
@@ -49,6 +50,11 @@ namespace Functions
    * want to evaluate your finite element function. In order to do so, it
    * needs to find out where the points lie.
    *
+   * @note This function is not intended for assembling a vector or a
+   * matrix on a cell-by-cell basis. Tailored access through FEValues
+   * or similar is recommended then. In such a case, the base class
+   * DoFVector may be helpful.
```

Comment:
I don't understand this comment. First, it's a class documentation, so what does "This function" refer to. Second, if the base class is useful, then it needs to be visible to the user -- but you use `private` inheritance.